src/veh_type.h

```diff
@@ -266,6 +266,9 @@ class vpart_info
         /** seatbelt (str), muffler (%), horn (vol), light (intensity) */
         int bonus = 0;
 
+        /** cargo weight modifier (percentage) */
+        int cargo_modifier = 100;
```

Comment:
The name of this should probably include the word "weight" as it is currently not clear from the code what exactly it modifies - the volume, the weight, the temperature?

Reply:
Maybe make that a float?

Reply:
I followed the logic of "damage_modifier". It's also percentage modifier.
At first I called it "cargo_weight_percent_factor". But that's probably too long.
Code guidelines said to avoid float so I did. 
Just tell me what you think will be the best and I will change it to that.

Reply:
Yes, I think that's okay to use integer for consistency. Just rename it to `cargo_weight_modifier` in code and json and it would be good to go.

Reply:
Ok, done that.